image/autodetect.go

```diff
@@ -67,46 +66,11 @@ func Autodetect(path string) (string, error) {
 		return TypeImage, nil
 
 	case "text/plain; charset=utf-8":
```

Comment:
Is there any reason autodetect can't be the default? This particular media type doesn't make sense. Media types don't actually have charsets (that is content types).